src/scanner/get-rule-inclusions.ts

```diff
@@ -24,6 +24,10 @@ export const explicitRuleOverrides: DictionaryStringTo<RuleIncluded> = {
         status: 'excluded',
         reason: 'Based on feedback from users we tested to check the user impact of duplicate ID failures on static elements. We found no user impact on the experience with any of the ATs including Narrator, JAWS, NVDA, Dragon, and Windows Speech Recognition. After further discussions with the Tooling Advisory Board, we decided to make this a best practice rule. See #4102.',
     },
+    'label-content-name-mismatch': {
+        status: 'included',
+        reason: 'for parity with video-caption, which axe-core includes by default',
```

Comment:
nit: reason to include this rule should be different (since the rule is not related with video-caption). Could be  something like " to include in Needs Review since its an experimental rule"